composer.json

```diff
@@ -27,7 +27,10 @@
         "preferred-install": {
             "*": "dist"
         },
-        "sort-packages": true
+        "sort-packages": true,
+        "allow-plugins": {
+            "symfony/flex": true
+        }
```

Comment:
@christinach I think adding just these lines to main might get it deployable again.  At least, that fixed the error message I got when deploying my branch.